data-protocols/ids/ids-api-multipart-dispatcher-v1/src/main/java/org/eclipse/dataspaceconnector/ids/api/multipart/dispatcher/sender/MultipartArtifactRequestSender.java

```diff
@@ -104,6 +104,8 @@ protected Message buildMessageHeader(DataRequest request, DynamicAttributeToken
                 ._requestedArtifact_(artifactId)
                 ._transferContract_(contractId)
                 .build();
+        request.getProperties().forEach((k, v) -> message.setProperty(k, v));
```

Comment:
The properties are attributed with `@JsonIgnore`. We should not use them, otherwise this functionally will be broken as soon as we serialize the Information Model class correctly.
Source:
https://github.com/International-Data-Spaces-Association/Java-Representation-of-IDS-Information-Model/blob/main/de/fraunhofer/iais/eis/ArtifactRequestMessageImpl.java

Reply:
since the INfomodel we be donated to EDC we're fine.